linux/LinuxProcessTable.c

```diff
@@ -78,11 +78,68 @@ static FILE* fopenat(openat_arg_t openatArg, const char* pathname, const char* m
    return fp;
 }
 
+static long fast_strtol_dec(char** str, int maxlen) {
+   long result = 0;
+   bool neg = false;
+
+   if (**str == '-') {
+      neg = true;
+      (*str)++;
+   }
+
+   if (!maxlen)
+      maxlen = 20; // −9223372036854775808
```

Comment:
If at all, document the why not the magic value …

And the why is basically "because C does not have default arguments" …